Makefile

```diff
@@ -71,3 +71,8 @@ test_all_commits_travis:
 	git config --global user.name "IPFS BOT"
 	git fetch origin master:master
 	GIT_EDITOR=true git rebase -i --exec "make test" master
+
+# since we have CI for osx and linux but not windows, this should help
+windows_build_check:
+	GOOS=windows GOARCH=amd64 go build -tags=nofuse ./cmd/ipfs
+	rm ipfs.exe
```

Comment:
would be nice to place it somewhere the user may not have one they use. lime use the output flag and put it in `.ipfs.exe` or something